src/util.py

```diff
@@ -0,0 +1,5 @@
+import discord
+
+def log_info_and_embed(embed: discord.Embed, logger, message: str):
+    embed.description += "\n" +  message
+    logger.info(message)
```

Comment:
im thinking we also add a TODO list here to send DMs to the user

Reply:
The intent with this method is to combine writing to logs and to the discord message we will send to the logging channel in one place. While we will need to be able to send a DM to the user, we won't always want to do that when we're logging things

Reply:
er mb i shouldve been clear, i meant at the end of the file to add a dm function

Reply:
Yeah, sending DMs makes sense to live in the utils file unless it needs a lot of weird interactions with discord objects

Reply:
I think it shouldnt need more than one since we should be able to just pass the discord user id string over and have discord process that on its side